code/jasmin/mlkem_avx2/kem.jinc

```diff
@@ -106,7 +106,7 @@ fn __crypto_kem_dec_jazz(reg u64 shkp, reg u64 ctp, reg u64 skp)
 
   buf[0:MLKEM_INDCPA_MSGBYTES] = __indcpa_dec_1(buf[0:MLKEM_INDCPA_MSGBYTES], ctp, skp);
 
-  hp = #LEA(skp + 32);
+  hp = skp; hp += 32;
   hp += 24 * MLKEM_K * MLKEM_N>>3;
```

Comment:
Can’t both additions be merged in a single one?

Reply:
Yes, they can

Reply:
https://github.com/formosa-crypto/libjade/commit/37c95022caaff934e131939d160f61832f39c302

Reply:
pushed a fix to hakyber